dbt_project.yml

```diff
@@ -29,20 +29,26 @@ vars:
     # snowplow__database: Only set if not using target.database for Snowplow events data -- WILL BE IGNORED FOR DATABRICKS
     # snowplow__events_table: "events" # Only set if not using 'events' table for Snowplow events data
     snowplow__events: "{{ source('atomic', 'events') }}"
-    snowplow__session_context: "{{ source('atomic','com_snowplowanalytics_snowplow_client_session_1') }}"
-    snowplow__mobile_context: "{{ source('atomic','com_snowplowanalytics_snowplow_mobile_context_1') }}"
-    snowplow__geolocation_context: "{{ source('atomic','com_snowplowanalytics_snowplow_geolocation_context_1') }}"
-    snowplow__application_context: "{{ source('atomic','com_snowplowanalytics_mobile_application_1') }}"
-    snowplow__screen_context: "{{ source('atomic','com_snowplowanalytics_mobile_screen_1') }}"
-    snowplow__app_errors_table: "{{ source('atomic','com_snowplowanalytics_snowplow_application_error_1') }}"
-    snowplow__screen_view_events: "{{ source('atomic','com_snowplowanalytics_mobile_screen_view_1') }}"
+    snowplow__session_context: com_snowplowanalytics_snowplow_client_session_1
+    snowplow__mobile_context: com_snowplowanalytics_snowplow_mobile_context_1
+    snowplow__geolocation_context: com_snowplowanalytics_snowplow_geolocation_context_1
+    snowplow__application_context: com_snowplowanalytics_mobile_application_1
+    snowplow__screen_context: com_snowplowanalytics_mobile_screen_1
+    snowplow__application_error_events: com_snowplowanalytics_snowplow_application_error_1
+    snowplow__screen_view_events: com_snowplowanalytics_mobile_screen_view_1
     snowplow__sessions_table: "{{ ref('snowplow_mobile_sessions') }}" # Change to your custom sessions table if you have disabled the standard sessions table in favour of a custom version. Advanced config.
     # Variables - Standard Config
     snowplow__start_date: "2020-01-01"
     snowplow__backfill_limit_days: 30
     snowplow__app_id: []
     snowplow__platform: ['mob']
     snowplow__session_stitching: true
+    snowplow__user_stitching_id: 'user_id'
+    snowplow__session_identifiers: [{'schema' : 'com_snowplowanalytics_snowplow_client_session_1', 'field' : 'session_id', 'prefix': 'sc'}]
```

Comment:
I think this will not work properly for Bigquery users out-of-the box as it doesn't combine versions, perhaps it is better to make it target based (_*).

Reply:
Same with the user identifier.

Reply:
Don't we need to change that in unified then 👀

Reply:
We already did it, that was the first bug fix straight after release.

Reply:
https://github.com/snowplow/dbt-snowplow-unified/blob/d7365445f5a7fd345dc083881d6485ad9439cb7e/macros/identifiers.sql#L21 ?

Reply:
I didn't scroll, ignore me...

Reply:
https://github.com/snowplow/dbt-snowplow-unified/blob/d7365445f5a7fd345dc083881d6485ad9439cb7e/macros/identifiers.sql#L64

Reply:
Yeah for now the intended route is to list each of the columns separately. But this is functionality that would definitely be included in a future release!